connectors/src/connectors/notion/temporal/workflows.ts

```diff
@@ -170,7 +174,7 @@ export async function notionSyncWorkflow(
                 parentClosePolicy:
                   ParentClosePolicy.PARENT_CLOSE_POLICY_TERMINATE,
               })
-            )
+            ) as Promise<SyncWorkflowResult>
```

Comment:
The pqueue adds a " | void" to the Promise return, I don't know why. For now I made a cast, feedback welcome